scripts/appleseedMaya/renderGlobals.py

```diff
@@ -431,48 +440,127 @@ def create(self):
+                            attrName="diffuseBounces")
+                        self._addControl(
+                            ui=pm.intFieldGrp(
+                                label="Glossy Bounces", numberOfFields=1, enable=limitBounces),
+                            attrName="glossyBounces")
+                        self._addControl(
+                            ui=pm.intFieldGrp(
+                                label="Specular Bounces", numberOfFields=1, enable=limitBounces),
+                            attrName="specularBounces")
+                        self._addControl(
+                            ui=pm.floatFieldGrp(
+                                label="Light Samples", numberOfFields=1),
+                            attrName="lightSamples")
+                        self._addControl(
+                            ui=pm.floatFieldGrp(
+                                label="Environment Samples", numberOfFields=1),
+                            attrName="envSamples")
+                        self._addControl(
+                            ui=pm.checkBoxGrp(label="Clamp Roughness"),
+                            attrName="clampRoughness")
+                        self._addControl(
+                            ui=pm.checkBoxGrp(label="Caustics"),
+                            attrName="caustics")
+                        self._addControl(
+                            ui=pm.floatFieldGrp(
+                                label="Max Ray Intensity", numberOfFields=1),
+                            attrName="maxRayIntensity")
+
+                with pm.frameLayout(label="Stochastic Progressive Photon Mapping", collapsable=True, collapse=False):
+                    with pm.columnLayout("appleseedColumnLayout", adjustableColumn=True, width=columnWidth):
+                        self._addControl(
+                            ui=pm.attrEnumOptionMenuGrp(
+                                label="Photon Type",
+                                enumeratedItem=self._getAttributeMenuItems("photonType")),
+                            attrName="photonType")
+                        self._addControl(
+                            ui=pm.attrEnumOptionMenuGrp(
+                                label="Direct Lighting",
+                                enumeratedItem=self._getAttributeMenuItems("SPPMLightingMode")),
+                            attrName="SPPMLightingMode")
+                        self._addControl(
+                            ui=pm.checkBoxGrp(label="Caustics"),
+                            attrName="SPPMCaustics")
+                        self._addControl(
+                            ui=pm.checkBoxGrp(label="Image Based Lighting"),
+                            attrName="SPPMEnableIBL")
+                        self._addControl(
+                            ui=pm.checkBoxGrp(
+                                label="Limit PT Bounces", changeCommand=self.__limitPhotonTracingBouncesChanged),
```

Comment:
If space allows, I would suggest omitting abbreviations: *Limit Photon Tracing Bounces* (I suppose that's what PT means here?)

Reply:
Same as above